ui/app/shared/locales/en/or.json

```diff
@@ -141,30 +141,25 @@
   "filterInvite": "Filter with at least 3 characters",
   "location": "Location",
   "longPressSetLoc": "Long press to set location",
+  "this": "Current",
+  "last": "Last",
+  "60minutes": "60 minutes",
   "hour": "Hour",
-  "lastHour": "Last hour",
-  "last24Hours": "Last 24 hours",
-  "thisHour": "This hour",
+  "6hours": "6 hours",
+  "24hours": "24 hours",
+  "7days": "7 days",
+  "30days": "30 days",
+  "90days": "90 days",
+  "6months": "6 months",
+  "365days": "365 days",
   "day": "Day",
-  "thisDay": "This day",
-  "thisDayLastWeek": "This day last week",
-  "yesterday": "Yesterday",
-  "last7Days": "Last 7 days",
-  "last30Days": "Last 30 days",
-  "last90Days": "Last 90 days",
```

Comment:
The strings that got removed  should also be removed from the other locales files.